### PR TITLE
added sv.xml and Swedish translations

### PR DESCRIPTION
**Pull request by felipufeliu**
I will finish this in a few hours.

### CODE REVIEW

File: src/main/resources/lang/sv.xml [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+        <classRestricted>Klassen {0} är begränsad</classRestricted>
+        <noClassChange>Du kan inte ändra klass eftersom din nuvarande klass är permanent.</noClassChange>
+        <classesDisabled>Klasser är inte aktiverade på den här kartan.</classesDisabled>
+        <noTeam>Du är inte i något lag.</noTeam>
+        <alreadyObs>Du är redan i det observerande laget.</alreadyObs>
+        <alreadyJoined>Du har redan gått med i {0}</alreadyJoined>
+        <teamFull>Beklagar, {0} är helt full</teamFull>
+        <forceExemption>{0} är befriad från att tvingas till ett lag.</forceExemption>
+        <noShuffle>Du kan inte blanda lagen under en pågående match.</noShuffle>
+        // I'm going to finish this then, actually I am a bit busy.
```
Comment (twizmwazin) [inline src/main/resources/lang/sv.xml]: XML comment syntax is

**Comment by twizmwazin** [comment]
I am going to wait until the translations are complete to merge this.

**Comment by felipufeliu** [comment]
Finished, this can be merged if you want now twiz

**Comment by iEli2tyree011** [comment]
Thanks felipe! :)

**Comment by twizmwazin** [comment]
:smile:

**Pull request merged**